x-pack/plugins/canvas/public/components/hooks/workpad/use_incoming_embeddable.ts

```diff
@@ -45,10 +49,19 @@ export const useIncomingEmbeddable = (selectedPage: CanvasPage) => {
       if (originalElement) {
         const originalAst = fromExpression(originalElement!.expression);
 
-        const functionIndex = originalAst.chain.findIndex(
-          ({ function: fn }) => fn === 'embeddable'
+        const functionIndex = originalAst.chain.findIndex(({ function: fn }) =>
+          ['embeddable', 'savedVisualization'].includes(fn)
         );
 
+        if (functionIndex === -1) {
+          return;
+        }
+
+        if (originalAst.chain[functionIndex].function === 'savedVisualization') {
+          notifyService.error(strings.getConvertToLensUnsupportedSavedVisualization());
```

Comment:
To resolve the elements stuck in loading, I think we want to always call `fetchAllRenderables()` when we return to Canvas which is one of the element actions, and it'll trigger all of the elements to reload.

```suggestion
          notifyService.error(strings.getConvertToLensUnsupportedSavedVisualization());
          dispatch(fetchAllRenderables());
```